index.php

```diff
@@ -33,7 +34,7 @@
         continue;
       }
 
-      $key = explode($server['seperator'], $key);
+      $key = explode($server['seperator'], $key); //@todo: may be separator ?
```

Comment:
I don't understand this comment?

Reply:
"Separator", not a "sepErator". Incorrect english word.

Reply:
It's not a bug ;-)

Reply:
🙈 too late to fix now I guess, it would break the configs people currently have.